### PR TITLE
PyOhio 2016 videos

### PR DESCRIPTION
**Pull request by mattboehm**
Added all video metadata and a category.json file.

The "alias" is based on my best guess from other files, but needs to be reviewed.

### CODE REVIEW

File: pyohio-2016/videos/sunday-lightning-talks.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "description": "",
+  "duration": 3600,
+  "recorded": "2016-07-31",
+  "speakers": [
+    ""
```
Comment (zerok) [inline pyohio-2016/videos/sunday-lightning-talks.json]: Speakers were as far as I can tell:

- ?
- Brandon Rhodes
- Joe Shaw
- Zach Wick
- Sam Kimbrel
- Qumisha Goss
- Cameron Blandford
- ?

**Comment by codersquid** [comment]
Hey y'all, if you use this url and then run the wget command at the top it will pull json files for all the pyohio videos in mostly the proper format http://veyepar3.nextdayvideo.com/main/M/pyvid_json.urls?client=pyohio&show=pyohio_2016

The thumbnail url isn't in these yet? but it turns out it is a fixed url with just the video id so you can snarf the json files to change that.

**Comment by codersquid** [comment]
I had originally started a branch to do this based on that but was waiting on Carl to clean up some of the metadata or something.

**Comment by mattboehm** [comment]
@codersquid the json files on that page seem to be for PyOhio 2015 for some reason.

It looks like the script I used to make the json files missed a few fields (copyright, slug, thumbnail_url, etc.) I can add some of these things to the current data by writing a one-off script, but would need to know what fields are critical.

**Comment by zerok** [comment]
@mattboehm Thanks for the fixes :)

@codersquid @mattboehm Sorry, totally missed the thumbnail_url field. That's IMHO the only essential one missing. If license/copyright information is available it would be a bonus :)

**Comment by codersquid** [comment]
It's CC-BY

**Comment by logston** [comment]
The `thumbnail_url` can be as simple as this: `https://i.ytimg.com/vi/<video id>/maxresdefault.jpg`

**Comment by logston** [comment]
Adding thumbnail urls to this branch and pr https://github.com/pyvideo/data/pull/106

**Pull request closed without merging**